.vscode/launch.json

```diff
@@ -10,7 +10,9 @@
             "type": "dart",
             "args": [
                 "--flavor",
-                "dev"
+                "dev",
+				"--dart-define",
```

Comment:
We don't need to use `--dart-define` (see the answer to the stackoverflow https://stackoverflow.com/questions/74751159/how-to-read-flavor-parameter-value-from-flutter-code)